### PR TITLE
Add Github CI on every PR

### PR DESCRIPTION
**Pull request by lpuv**
Adds Github CI to every PR.

The CI automagically runs a native Debian build and a build using makedeb itself.

### CODE REVIEW

**Comment by lpuv** [comment]
AHH IT FINALLY WORKS

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+on:
+ pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      TARGET: local
+      RELEASE: alpha
+#    container:
+#      image: sardine123/makedeb-github-ci:ubuntu-focal
```
Comment (hwittenborn) [inline .github/workflows/pr.yml]: Is there any use in keeping these comments here?

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+#      - name: Setup version env var
+#        run: |
+#         cat << EOF > ../.env
+#            VERSION=$(cat .data.json | jq -r '.current_pkgver')
+#          EOF
```
Comment (hwittenborn) [inline .github/workflows/pr.yml]: Same here

File: .github/workflows/pr.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        run: |
+          git tag -f "v$(cat .data.json | jq -r '.current_pkgver + "-" + .current_pkgrel')-alpha"
+          sudo apt update && sudo apt install -y git jq
+          cd PKGBUILD
+          ./pkgbuild.sh > PKGBUILD
+          PACMAN='/usr/bin/true' makedeb -s --no-confirm
+          ls -alt
+          mv -v makedeb-alpha_$(cat ../.data.json | jq -r '.current_pkgver + "-" + .current_pkgrel')_all.deb ../makedeb.deb
+      #- name: Rename result
+      #  run: source .env && mv makedeb-alpha_$VERSION-1_all.deb makedeb.deb
```
Comment (hwittenborn) [inline .github/workflows/pr.yml]: Same here

**Review by hwittenborn — APPROVED** [review]
LGTM, hoping it works :)

**Pull request merged**